musl build fixes, escape.c calls nl_langinfo() without including langinfo.h

--- a/library/escape.c
+++ b/library/escape.c
@@ -23,6 +23,7 @@
 #include <limits.h>
 #include <stdio.h>
 #include <string.h>
+#include <langinfo.h>
 
 #include "escape.h"
 #include "readproc.h"
